libethereum/State.cpp

```diff
@@ -342,6 +342,14 @@ void State::noteSending(Address const& _id)
 		a->incNonce();
 }
 
+void State::revertIncNonce(Address const& _addr)
+{
+	auto& nonce = account(_addr)->nonce();
+	cwarn << "Revert nonce " << _addr << nonce;
```

Comment:
Too defensive in this case. I know it has been increased. Anyway, I'm working on improved account nonce API.